Subsurface/FlightComputer/CMakeLists.txt

```diff
@@ -1,12 +1,15 @@
-cmake_minimum_required(VERSION 3.13)
+cmake_minimum_required(VERSION 3.15)
```

Comment:
Had to bump this so that I could properly set the MSVC runtime library below (that property was introduced in 3.15). If this is an issue then lmk but I believe the old version of cmake on apt could be avoided by installing through pip

Reply:
I usually use pip to get cmake anyways

Reply:
Can confirm pip cmake ran correctly